### PR TITLE
Remove native Statusbar and Titlebar from Templates (#4671 )

### LINKED ISSUE #4671: [Hybrid] Handling Safearea, Statusbar and Titlebar in project templates
For handling Safearea, Statusbar and Titlebar in project templates we need to do a couple of steps:

- [x] Create platform specific css classes
- [x] Remove Titlebar and Statusbar from MAUI projects
- [x] Handle Statusbar foreground color
- [x] Adapt TodoTemplate's UI with Web based Statusbar/Titlebar
- [x] Adapt AdminPanel Template's UI with Web based Statusbar/Titlebar
- [x] Fix the remain Bit BlazorUI demo UI issues

### PR DESCRIPTION
**Pull request by mhrastegari**
Closes #4671

### CODE REVIEW

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/App.razor.cs [hunk 1]
```diff
@@ -13,11 +14,89 @@ public partial class App
...
+
+        await base.OnAfterRenderAsync(firstRender);
+    }
+#endif
+
+    private async Task SetupBodyClasses()
+    {
+        var cssClasses = new List<string>();
+
+        if (BlazorModeDetector.Current.IsBlazorWebAssembly())
```
Comment (ysmoradi) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/App.razor.cs:48]: if blazor electron is missing

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts [hunk 2]
```diff
@@ -22,3 +22,8 @@
         document.body.style.overflow = hidden ? "hidden" : "auto";
     }
 }
+
+function applyBodyElementClasses(cssClasses: string[], cssVariables: any) {
```
Comment (ysmoradi) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts:26]: Move this to App class

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Styles/app.scss [hunk 3]
```diff
@@ -1,5 +1,6 @@
 * {
     overscroll-behavior: none;
+    font-family: "Segoe UI";
```
Comment (ysmoradi) [inline src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Styles/app.scss:3]: is this font pre-loaded?

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts (hunk 2 above)
Comment (mhrastegari) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts:26]: Check the line 83 :")

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/App.razor.cs (hunk 1 above)
Comment (mhrastegari) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/App.razor.cs:48]: Check the line 83 :")

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Styles/app.scss (hunk 3 above)
Comment (mhrastegari) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Styles/app.scss:3]: Yes :)

File: src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts (hunk 2 above)
Comment (mhrastegari) [reply src/Templates/AdminPanel/Bit.AdminPanel/src/Client/Core/Scripts/app.ts]: Fixed it in the latest commit

**Review by msynk — APPROVED** [review]

**Pull request merged**